modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2927,124 +2927,138 @@ definition {
+				ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
 
-			CreateObject.selectTitleField(fieldLabel = "Custom Field ${letter}");
+				SelectFrame(locator1 = "IFrame#IFRAME");
 
-			PortletEntry.publish();
+				Click.clickAtNotVisible(locator1 = "ObjectAdmin#ENTRY_BOOLEAN_CHECKBOX");
 
-			ObjectAdmin.goToLayoutsTab();
+				ObjectAdmin.goToLayoutTabOnLayouts();
 
-			WaitForElementPresent(locator1 = "ObjectAdmin#KEBAB_MENU");
+				ObjectAdmin.addTabFieldsOnLayout(
+					blockName = "Block 1",
+					columnsNumber = 2,
+					fieldLabelName = "Custom Field ${letter}",
+					tabName = "Field Tab");
 
-			ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
+				ObjectAdmin.addTabRelationshipsOnLayout(
+					relationshipsLabelName = "Relationship",
+					tabName = "Relationship Tab");
 
-			SelectFrame(locator1 = "IFrame#IFRAME");
+				PortletEntry.save();
+			}
+		}
 
-			Click.clickAtNotVisible(locator1 = "ObjectAdmin#ENTRY_BOOLEAN_CHECKBOX");
+		task ("And given the Custom Field is defined as a title field in the custom objects") {
+			for (var letter : list "A,B") {
+				ObjectAdmin.openObjectAdmin();
 
-			ObjectAdmin.goToLayoutTabOnLayouts();
+				ObjectPortlet.selectCustomObject(label = "Custom Object ${letter} 74");
 
-			ObjectAdmin.addTabFieldsOnLayout(
-				blockName = "Block 1",
-				columnsNumber = 2,
-				fieldLabelName = "Custom Field ${letter}",
-				tabName = "Field Tab");
+				ObjectAdmin.goToDetailsTab();
 
-			ObjectAdmin.addTabRelationshipsOnLayout(
-				relationshipsLabelName = "Relationship",
-				tabName = "Relationship Tab");
+				CreateObject.selectTitleField(fieldLabel = "Custom Field ${letter}");
 
-			PortletEntry.save();
+				PortletEntry.publish();
 
-			Navigator.openURL();
+				Navigator.openURL();
+			}
 		}
 
-		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObjectA74");
+		task ("When multiple entries are created for the custom objects") {
+			for (var letter : list "A,B,C") {
+				ObjectAdmin.goToCustomObject(objectName = "CustomObjectA74");
 
-		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObjectB74");
+				ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry ${letter}");
+			}
 
-		for (var letter : list "A,B,C") {
-			ObjectAdmin.goToCustomObject(objectName = "CustomObjectA74");
-
-			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry ${letter}");
-		}
-
-		Navigator.openURL();
+			Navigator.openURL();
 
-		for (var letter : list "D,E,F") {
-			ObjectAdmin.goToCustomObject(objectName = "CustomObjectB74");
+			for (var letter : list "D,E,F") {
+				ObjectAdmin.goToCustomObject(objectName = "CustomObjectB74");
 
-			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry ${letter}");
+				ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry ${letter}");
+			}
 		}
 
-		Navigator.openURL();
+		task ("Then multiple object entries can be related on both objects") {
+			Navigator.openURL();
```

Comment:
Skip this step since we're going to the object page right after. No need to go to the home page here.